source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/direct/factory/torch_jit_utils.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import torch
+
+import omni.isaac.core.utils.torch as torch_utils
+
+"""
+Rotation conversions
+"""
+
+
+@torch.jit.script
+def quat_to_angle_axis(q):
```

Comment:
we should have a similar `axis_angle_from_quat` in math.py

Reply:
Replaced and removed this script